### PR TITLE
re-design of members, offers & inquires

### PR DESCRIPTION
**Pull request by zarpilla**
Hi, 

a first redesign of the members, offers & inquires based on the wireframes from 
https://community.coopdevs.org/t/wireframes-per-mobile-nova-estrategia/933
to solve [the issue](https://github.com/coopdevs/timeoverflow/issues/496)

### CODE REVIEW

File: app/assets/stylesheets/_to-member-card.scss [hunk 1]
```diff
@@ -1,84 +1,173 @@
-.to-member-card {
+.to-member-card, .panel-default, .simple_form {
   & {
-    margin: 16px 0;
+    margin: 24px 0;
     width: 100%;
-    max-width: 600px;
-    height: 178px;
+    //max-width: 600px;
+    //height: 178px;
```
Comment (enricostano) [inline app/assets/stylesheets/_to-member-card.scss]: Do we need these variables?

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -338,14 +365,16 @@ label[required]::after{
...
   border: 0;
   background-color: transparent;
   padding: 0;
 
-  @media(max-width: $screen-sm-min) {
-    padding: 10px 15px;
-  }
+  // @media(max-width: $screen-sm-min) {
+  //   padding: 10px 15px;
+  // }
```
Comment (enricostano) [inline app/assets/stylesheets/application.css.scss]: Is this code needed?

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -373,12 +402,17 @@ label[required]::after{
...
-  background-color: rgba($palette-turkey, 0.9);
+  &.mdc-button{
+    margin:0;
+    margin-top:5px;
+  }
 }
 
+//.navbar-static-top {
+  //background-color: $palette2-grey;
+//}
```
Comment (enricostano) [inline app/assets/stylesheets/application.css.scss]: Is this code needed?

File: app/views/shared/_show_filter_hint.html.erb [hunk 4]
```diff
@@ -1,18 +1,18 @@
 <% if @category.present? %>
   <small>
-    <%= glyph :category %>
+    <!-- %= glyph :category % -->
```
Comment (enricostano) [inline app/views/shared/_show_filter_hint.html.erb]: Is this needed?

File: app/views/shared/_show_filter_hint.html.erb [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
     <%= @category.name %>
   </small>
 <% end %>
 <% if params[:q].present? %>
   <small>
-    <%= glyph :search %>
+  <!-- %= glyph :search % -->
```
Comment (enricostano) [inline app/views/shared/_show_filter_hint.html.erb]: Is this needed?

File: app/views/shared/_post.html.erb [hunk 6]
```diff
@@ -1,74 +1,75 @@
...
     <% if current_user && current_organization == post.organization %>
-      <div class="panel panel-info">
-        <div class="panel-heading">
-          <h3 class="panel-title">
-            <%= link_to post.member.display_name_with_uid, post.user  %>
+      <div class="panel panel-default">
+        <div class="panel-heading panel-heading-bg">
+          <h3 class="panel-title panel-body-with-avatar">
+          <%= link_to image_tag(avatar_url(post.user, 36)), post.user %>
+            <%= link_to post.member, post.user  %>
```
Comment (enricostano) [inline app/views/shared/_post.html.erb]: Is this indentation correct?

File: app/views/shared/_show_filter_hint.html.erb [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
     <%= params[:q] %>
   </small>
 <% end %>
 <% if params[:tag].present? %>
   <small>
-    <%= glyph :tag %>
+    <!-- %= glyph :tag % -->
```
Comment (enricostano) [inline app/views/shared/_show_filter_hint.html.erb]: Is this needed?

File: app/views/users/_member_card.html.erb [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
+<div class="col-lg-6 col-md-6 col-sm-12">
   <div class="to-member-card">
     <div class="to-member-card__header">
-      <div class="to-member-card__header__avatar"><%= member.avatar_img(48) %></div>
+      <div class="to-member-card__header__avatar"><%= member.avatar_img(58) %></div>
       <div class="to-member-card__header__text">
         <h4><%= member.link_to_self %></h4>
         <div class="to-member-card__header__text__activity">
-          <%= "##{member.member_uid} — " %>
+          <!-- <%= "##{member.member_uid} — " %> -->
```
Comment (enricostano) [inline app/views/users/_member_card.html.erb]: Is this needed?

File: app/views/users/show.html.erb [hunk 9]
```diff
@@ -73,32 +194,35 @@
       </div>
     </div>
   </div>
-</div>
+</div-->
```
Comment (enricostano) [inline app/views/users/show.html.erb]: Is this needed?

File: app/views/users/show.html.erb [hunk 10]
```diff
@@ -166,6 +290,6 @@
...
```
Comment (enricostano) [inline app/views/users/show.html.erb]: Is this needed?

**Comment by markets** [comment]
@zarpilla @sseerrggii @enricostano 👋 I think we can close this one as it's superseded by #501

**Pull request closed without merging**